boggle.js

```diff
@@ -1,26 +1,45 @@
 function newBoggleBoard() {
-  return [
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-  ];
+
+  let boggleBoard = [];
```

Comment:
What's the motivation for changing it from the skeleton code? This works, but not sure if you think it does something different.